sdk/src/main/java/co/omise/android/api/Invocation.kt

```diff
@@ -0,0 +1,78 @@
+package co.omise.android.api
+
+import android.os.Handler
+import co.omise.android.models.APIError
+import co.omise.android.models.Model
+import co.omise.android.models.ModelParserUtil.parseModelFromJson
+import okhttp3.Call
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.Response
+import org.json.JSONException
+import java.io.IOException
+
+internal class Invocation<T : Model>(
+        private val replyHandler: Handler,
+        private val httpClient: OkHttpClient,
+        private val request: co.omise.android.api.Request<T>,
+        private val listener: RequestListener<T>
+) {
+
+    fun invoke() {
+        val call = httpClient.newTypedCall(
+                Request.Builder()
+                        .method(request.method,
```

Comment:
Should we make it same line?

Reply:
Which ones?

Reply:
`method()`

Reply:
Hmmm, alright 👌

Reply:
@nuxzero how about this? 

`
        val call = httpClient.newTypedCall(Request.Builder()
                .method(request.method,
                        request.payload)
                .url(request.url)
                .build(),
                request.responseType)
`

Reply:
@nuxzero How about this? 

![E5C70AAD-F1E5-401D-B55F-D5075FD15CE5](https://user-images.githubusercontent.com/5729822/60663279-ffb52500-9e73-11e9-8ea0-8fe7e1a6a5cf.jpg)

Reply:
No no, I mean 
```
val call = httpClient.newTypedCall(
                Request.Builder()
                        .method(request.method, request.payload)
                        .url(request.url)
                        .build(),
                request.responseType)
```

Reply:
BTW, approved. 👍

Reply:
Ahhhh okk, now I get it 👌